setup.py

```diff
@@ -90,6 +90,10 @@
     "pytz",
     "characteristic >= 14.1.0",
     "Twisted == 15.1.0",
+    # TLS support libraries for Twisted:
+    "service_identity == 14.0.0",
```

Comment:
Is it more important with these dependencies than others that they are kept up to date (just because they relate to security)?